modules/ROOT/pages/aurads/tutorials/arrow-examples.adoc

```diff
@@ -40,18 +40,13 @@ gds = GraphDataScience(NEO4J_CONNECTION_URI, auth=(NEO4J_USERNAME, NEO4J_PASSWOR
 gds.set_database("neo4j")
 ----
 
-The `gds.debug.arrow()` procedure can be used to verify that Arrow is enabled and running.
-
-[source, python]
-----
-gds.run_cypher("CALL gds.debug.arrow() YIELD enabled, running")
-----
+You can call the `gds.debug.arrow()` procedure to verify that Arrow is enabled and running.
```

Comment:
if we remove the example query below how about:
```suggestion
You can run `CALL gds.debug.arrow()` procedure to verify that Arrow is enabled and running.
```

Reply:
Since we are only showing the client here, I would not add Cypher procedures to the mix - so maybe it's better to bring back the example?